README.md

```diff
@@ -1,8 +1,27 @@
 # kubectl-cost
 
-`kubectl-cost` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) that provides easy CLI access to Kubernetes cost allocation metrics via the [kubecost APIs](https://github.com/kubecost/docs/blob/master/apis.md). It allows developers, devops, and others to quickly determine the cost & efficiency for any Kubernetes workload.
+`kubectl-cost` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/)
+that provides easy CLI access to Kubernetes cost information, like historical cost
+and predicted future cost, via [Kubecost's APIs](https://docs.kubecost.com/). It
+allows developers, devops, and others to quickly determine the cost & efficiency
+of any Kubernetes workload.
+
+> If you use [OpenCost](https://github.com/opencost/opencost), most of 
+> `kubectl cost` works! See the [OpenCost documentation](https://www.opencost.io/docs/kubectl-cost)
+> for examples. Let us know how it goes, and open an issue if you encounter any
+> problems!
+
+‼️ New! Starting with Kubecost v1.100, `kubectl cost` can estimate the cost of undeployed changes! Try it
```

Comment:
I'd really like to have some color here, rather than just bolding. There's a lot to be said for a GitHub README that pops on first view, which is why we already have some GIFs. Do you think there's a way we could incorporate some eye-catching color (or some other mechanism) to draw attention to the new feature?

Reply:
Unless we want to use an image file (.png or .gif), which would be the most immediate implementation, we may have to look at third party plugins. I haven't implemented one of these in the past, but this should have what you're looking for:
https://docs.gitlab.com/ee/user/markdown.html

Reply:
What I mean to say is: emojis are an easy way to add color that are supported by default in GitHub markdown, which is why I went for some sort of emoji. Given that a README is half marketing landing page, half documentation, is an emoji inappropriate?

Reply:
Ah, I wasn't aware that was supposed to be an emoji. In general I would say we should avoid them in tech docs, but there are existing exceptions like on our 3-step Kubecost install page which I've received pushback on weirdly. For now, I'm just going to table this and try to find a consistent system of using these "banners" across our docs, just as in warnings.